packages/account/src/Components/sent-email-modal/sent-email-modal.jsx

```diff
@@ -104,7 +104,6 @@ const SentEmailModal = ({
             txt_resend={localize('Resend email')}
             txt_resend_in={localize('Resend email in')}
             onClickSendEmail={onClickSendEmail}
-            closeEmailModal={onClose}
```

Comment:
We don't need this?

Reply:
@niloo-fs we don't have it in `send-email-template.tsx`

Reply:
thanks @iman-fs 
yeas, I didn’t find its usage